graql/reasoner/atom/task/convert/RelationAtomConverter.java

```diff
@@ -84,7 +84,7 @@ public AttributeAtom toAttributeAtom(RelationAtom atom, ReasoningContext ctx) {
 
     @Override
     public IsaAtom toIsaAtom(RelationAtom atom, ReasoningContext ctx) {
-        IsaAtom isaAtom = IsaAtom.create(atom.getVarName(), atom.getPredicateVariable(), atom.getTypeLabel(), false, atom.getParentQuery(), ctx);
+        IsaAtom isaAtom = atom.isaAtom();
```

Comment:
Why can't we just return an isaAtom here? Converting a RelationAtom with an `isa` in it into an `isaAtom` can only contain the information that `X isa RelationType`, and nothing else, right?